### PR TITLE
Cherry-picks for the 4.1 branch (future 4.1.1) - 1st batch

### PR DESCRIPTION
**Pull request by YuriSizov**
Not including changelog for now.

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]
Looks good to me. Pre-emptively approving, though I'm aware Yuri will add a couple cherry-picks from PRs merged today (also approved by me).

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,168 @@ All notable changes to this project will be documented in this file.
...
+- Use bullet points in shader editor creation dialog ([GH-78631](https://github.com/godotengine/godot/pull/78631)).
+- Sort project tags before saving ([GH-78775](https://github.com/godotengine/godot/pull/78775)).
+- Project converter: Use same rendering driver as Project Manager ([GH-78795](https://github.com/godotengine/godot/pull/78795)).
+- Add tooltip description wrapping in scene tree and plugin settings ([GH-79090](https://github.com/godotengine/godot/pull/79090)).
+- Improve user-friendliness of project version mismatch messages ([GH-79118](https://github.com/godotengine/godot/pull/79118), [GH-79299](https://github.com/godotengine/godot/pull/79299)).
+
+#### GUI
+
+- Ensure that `_drop_physics_mouseover` only happens when necessary ([GH-78078](https://github.com/godotengine/godot/pull/78078)).
+- Update FileDialog button activity when file_mode is changed ([GH-79211](https://github.com/godotengine/godot/pull/79211)).
```
Comment (akien-mga) [inline CHANGELOG.md:61]:
```suggestion
- Update FileDialog button activity when `file_mode` is changed ([GH-79211](https://github.com/godotengine/godot/pull/79211)).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,168 @@ All notable changes to this project will be documented in this file.
...
+
+- Fix zero-sized WorkerThreadPool not processing group tasks ([GH-78845](https://github.com/godotengine/godot/pull/78845)).
+- Fix `Node::add_sibling` parent check ([GH-78847](https://github.com/godotengine/godot/pull/78847)).
+- Fix error when non-ASCII characters in resource pack path ([GH-78935](https://github.com/godotengine/godot/pull/78935)).
+- Fix erroneous `pad_zeros()` warning ([GH-79202](https://github.com/godotengine/godot/pull/79202)).
+- Fix `PackedScene::get_last_modified_time()` always returns `0` ([GH-79237](https://github.com/godotengine/godot/pull/79237)).
+
+#### Editor
+
+- Fix dropping files from res:// to res:// ([GH-78914](https://github.com/godotengine/godot/pull/78914)).
```
Comment (akien-mga) [inline CHANGELOG.md:107]:
```suggestion
- Fix dropping files from `res://` to `res://` ([GH-78914](https://github.com/godotengine/godot/pull/78914)).
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,168 @@ All notable changes to this project will be documented in this file.
...
+
+- Fix threading bug in Vulkan rendering device ([GH-78794](https://github.com/godotengine/godot/pull/78794)).
+- Fix sanitizers reports about octahedral tangents in RenderingServer ([GH-78902](https://github.com/godotengine/godot/pull/78902)).
+
+#### Shaders
+
+- Fix invalid shader compilation when using `hint_normal_roughness_texture` in mobile backend ([GH-78839](https://github.com/godotengine/godot/pull/78839)).
+- Fix using uint suffix at the hex number declaration in shaders ([GH-78906](https://github.com/godotengine/godot/pull/78906)).
+- Fix shader language float literal precision truncation ([GH-78972](https://github.com/godotengine/godot/pull/78972)).
+- Fix comments and indentation in .gdshaderinc files ([GH-79158](https://github.com/godotengine/godot/pull/79158)).
```
Comment (akien-mga) [inline CHANGELOG.md:166]:
```suggestion
- Fix comments and indentation in `.gdshaderinc` files ([GH-79158](https://github.com/godotengine/godot/pull/79158)).
```

**Review by akien-mga — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,168 @@ All notable changes to this project will be documented in this file.
...
+
+### Added
+
+#### Animation
+
+- Add 3.x compatibility for animation loop mode ([GH-79155](https://github.com/godotengine/godot/pull/79155)).
+
+#### C#/.NET
+
+- Automatically generate version defines ([GH-78722](https://github.com/godotengine/godot/pull/78722)).
```
Comment (RedworkDE) [inline CHANGELOG.md:19]: This is just a buildsystem internal change that has no effect on users compared to a PR manually updating the version, so I wouldn't feature this so prominently, if at all.
Ie for users this is basically on the same level as increasing a version number.

**Pull request merged**